libs/blocks/icon-block/icon-block.css

```diff
@@ -164,6 +164,14 @@
   margin-bottom: var(--spacing-s);
 }
 
+[dir="rtl"] .icon-block .foreground {
+  text-align: right;
+}
+
+[dir="rtl"] .icon-block .foreground .icon-area {
+  text-align: center;
+}
```

Comment:
Hi, I added that because of this screenshoot of old page that was attached to my ticket
<img width="1231" alt="rtl" src="https://github.com/adobecom/milo/assets/90400759/792fad1c-179d-414e-a4db-785d8d16262a"> 
If this is a big issue I will talk to my team and see if it is okay that icons are aligned to the right?

Reply:
I agree with @overmyheadandbody - makes sense to keep the alignment consistent.